client/python/ovmsclient/lib/ovmsclient/tfs_compat/http/requests.py

```diff
@@ -187,14 +194,14 @@ def _parse_input_data(values):
     else:
         raise TypeError("provided values type is not valid")
 
-    if dtype == DataType.DT_STRING and _is_bytes_shape_valid(tensor_values.shape, tensor_values):
-        raise ValueError("bytes values with dtype DT_STRING must be in shape [N]")
-
-    if dtype == DataType.DT_STRING:
-        b64_values = []
-        for value in tensor_values:
-            b64_value = base64.b64encode(value).decode('utf-8')
-            b64_values.append({"b64": b64_value})
-        return b64_values
-    else:
-        return tensor_values.tolist()
+    if dtype == DataType.DT_STRING and tensor_values.dtype.type == np.bytes_:
+        if len(tensor_values.shape) != 1:
+            # Bytes in multidimensional shape will be encoded in UTF-8 as regular input
+            tensor_values = _decode_bytes_to_strings(tensor_values)
+        else:
+            b64_values = []
+            for value in tensor_values:
+                b64_value = base64.b64encode(value).decode('utf-8')
+                b64_values.append({"b64": b64_value})
+            return b64_values
+    return tensor_values.tolist()
```

Comment:
Add example how to pack single string (and multiple strings) into request and how to read string (or multiple strings) from results